src/App.js

```diff
@@ -0,0 +1,19 @@
+import { Routes, Route } from 'react-router-dom';
+import Header from './Components/Header';
+import Home from './Components/Home';
+import Items from './Components/Itemspage';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <Header />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/items" element={<Items />} />
+      </Routes>
```

Comment:
As mentioned in the [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Every%20page%20(the%20main%20page%20and%20the%20pages%20for%20each%20item)%20should%20have%20a%20unique%20route%20within%20the%20SPA.), every page (the main page and the pages for each item) should have a unique route within the SPA. All details pages have the same route. Each page should have its own individual route. A way to fix this would be to add the item's id to it like this: 
```
<Route path="/items/:id" element={<Items />} />
```

Reply:
@laiifuu i have doubt about optional suggestion. The api i used dont give all data even 1 page data is about 4000 lines so even if i fecth all the data it would take a lot of time to first load. i can limit fetch request for detailpage as i already have details data because i already fetched it in homepage. what would be better here In api there's already query for seach anime and pages depending on url so its better to not fetch all the data as it would take a lot of time to load that much data. This is api link https://docs.api.jikan.moe/. what would be best approach here?.

I will implement second requirement right away.